test/bitcoin/scriptTest.mo

```diff
@@ -0,0 +1,58 @@
+import Script "../../src/bitcoin/Script";
+import Curves "../../src/ec/Curves";
+import PublicKey "../../src/ecdsa/Publickey";
+import Debug "mo:base/Debug";
+import TestUtils "../TestUtils";
+
+type P2pkhTestCase = {
+  publicKey : [Nat8];
+  expectedBytes : [Nat8];
+};
+
+let p2pkhTestCases : [P2pkhTestCase] = [
+  {
+    publicKey = [
+      0x03, 0x9a, 0xc8, 0xba, 0xc8, 0xf6, 0xd9, 0x16, 0xb8, 0xa8, 0x5b, 0x45,
+      0x8e, 0x08, 0x7e, 0x0c, 0xd0, 0x7e, 0x6a, 0x76, 0xa6, 0xbf, 0xdd, 0xe9,
+      0xbb, 0x76, 0x6b, 0x17, 0x08, 0x6d, 0x9a, 0x5c, 0x8a
+    ];
+    expectedBytes = [
```

Comment:
How did you derive the expected bytes?
Obviously, it is not clear for a reviewer that these bytes constitute the correct result.

Reply:
Moved to [p2pkhTest.mo](https://github.com/tgalal/motoko-bitcoin/pull/7/files#diff-933d1a0f2cc0d554d04e632b6b5a08abdcddd310bbc5cc2694e16262a866a51c) and added source of data in a comment.